pulsar-functions/instance/src/main/java/org/apache/pulsar/functions/instance/InstanceUtils.java

```diff
@@ -173,4 +176,25 @@ public static PulsarClient createPulsarClient(String pulsarServiceUrl, Authentic
         }
         return null;
     }
+
+    public static PulsarAdmin createPulsarAdminClient(String pulsarWebServiceUrl, AuthenticationConfig authConfig) throws PulsarClientException {
+        PulsarAdminBuilder pulsarAdminBuilder = null;
+        if (isNotBlank(pulsarWebServiceUrl)) {
+            pulsarAdminBuilder = PulsarAdmin.builder().serviceHttpUrl(pulsarWebServiceUrl);
+            if (authConfig != null) {
+                if (isNotBlank(authConfig.getClientAuthenticationPlugin())
+                        && isNotBlank(authConfig.getClientAuthenticationParameters())) {
+                    pulsarAdminBuilder.authentication(authConfig.getClientAuthenticationPlugin(),
+                            authConfig.getClientAuthenticationParameters());
+                }
+                if (isNotBlank(authConfig.getTlsTrustCertsFilePath())) {
+                    pulsarAdminBuilder.tlsTrustCertsFilePath(authConfig.getTlsTrustCertsFilePath());
+                }
+                pulsarAdminBuilder.allowTlsInsecureConnection(authConfig.isTlsAllowInsecureConnection());
+                pulsarAdminBuilder.enableTlsHostnameVerification(authConfig.isTlsHostnameVerificationEnable());
+            }
+            return pulsarAdminBuilder.build();
+        }
+        return null;
```

Comment:
since the method throws an exception, it might be worth throwing an exception saying the `pulsarWebServiceUrl` is null.